src/core/client/admin/components/UserRole/UserRoleChangeContainer.tsx

```diff
@@ -38,11 +38,6 @@ const UserRoleChangeContainer: FunctionComponent<Props> = ({
   );
   const handleOnChangeRole = useCallback(
     async (role: GQLUSER_ROLE_RL, siteIDs?: string[]) => {
-      if (role === user.role) {
```

Comment:
I considered adding an check to see if in the case of matching roles, the roles were "MODERATOR" and all the siteIDs were the same as the user's current site ids but decided to just remove this check on the frontend because it is also handled on the backend and makes the code more understandable. Let me know if you disagree!

Reply:
As long as we're still preventing them from clicking on "change role" via disabling/hiding options (which I believe we are) this should be fine.